android/java/org/chromium/chrome/browser/preferences/BraveMainPreferences.java

```diff
@@ -0,0 +1,64 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.preferences;
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.preference.Preference;
+
+import java.util.HashMap;
+
+// This exculdes some settings in main settings screen.
+public class BraveMainPreferences extends MainPreferences {
```

Comment:
I'm thinking whether super class is better or subclass is better for customizing main preferences.

Reply:
Changed to use super class. With this, we don't need to worry about the code that use `MainPreferences.class` directly.